submissions/2025-03-12-SPICE-DES370k-Monomers-Partial-Relaxation-Dataset-v4.0/README.md

```diff
@@ -0,0 +1,57 @@
+# SPICE DES370k Monomers Partial Relaxation Dataset v4.0
+
+## Description
```

Comment:
According to [v4.0 standards](https://github.com/openforcefield/qca-dataset-submission/blob/master/STANDARDS.md) the following information needs to be here (and in the dataset assembled in the Jupyter notebook).:
Intended usage of the dataset
Level of theory
Elements covered
Charges covered
Min, mean, and max molecular weight of the molecules in the dataset